src/services/resources-getter.js

```diff
@@ -8,17 +8,50 @@ import LiveQueryChecker from './live-query-checker';
 import { ErrorHTTP422 } from './errors';
 import FiltersParser from './filters-parser';
 import extractRequestedFields from './requested-fields-extractor';
+import ScopeProvider from './scope-provider';
 
+
+/**
+ * An instance of this class represents a subset of a given collection.
+ * i.e. All books with a red cover.
+ *
+ * The two public methods are perform() and count()
+ * All the rest can be considered private
+ */
 class ResourcesGetter {
-  constructor(model, options, params) {
+  /**
+   * FIXME
+   * AFAIK lianaOptions is a global. Why pass it as param everywhere?
```

Comment:
[MISC] If so, it could be great to get rid of it, but it looks a bit out of scope for now